### PR TITLE
fix: React Wallet uses search without query (fixes #165)

### LINKED ISSUE #165: React wallet sample getting unresponsive and not working.
- I am using react wallet sample as a reference to integrate functionalities in my web-app, but the sample app is not working and I can't map the workings.
- The error I am getting in console, 
image

@janpieterz

### PR DESCRIPTION
**Pull request by janpieterz**

### CODE REVIEW

File: browser/react-wallet/src/actions/index.ts [hunk 1]
```diff
@@ -152,10 +150,8 @@ export function getWalletItems(): ThunkAction<
     getState: () => ActionState
   ) => {
     setAuthTokenFromState(getState);
-    let response = await service.wallet().search(SearchRequest.fromPartial({}));
-    // let items = response.getItemsList().map(item => item.getJsonStruct().toJavaScript());
-    // items = items.map(item => JSON.parse(Object.values(item.data).join('')))
-    let items = response.items.map((item) => JSON.parse(JSON.parse(item).data));
```
Comment (janpieterz) [inline browser/react-wallet/src/actions/index.ts:158]: @fundthmcalculus did the response type of this change?

Comment (fundthmcalculus) [reply browser/react-wallet/src/actions/index.ts:158]: Not to the extent of my knowledge.

Comment (janpieterz) [reply browser/react-wallet/src/actions/index.ts:158]: Odd. This works, confirmed with a result from the search, so I guess it's good. 

@MichaelEdwardBlack or @tmarkovski any idea?

**Review by fundthmcalculus — APPROVED** [review]

File: browser/react-wallet/src/actions/index.ts [hunk 2]
```diff
@@ -152,10 +150,8 @@ export function getWalletItems(): ThunkAction<
     getState: () => ActionState
   ) => {
     setAuthTokenFromState(getState);
-    let response = await service.wallet().search(SearchRequest.fromPartial({}));
```
[the remark this answers is not in the document]
Comment (janpieterz) [reply browser/react-wallet/src/actions/index.ts:155]: Awesome

**Pull request merged**